### ISSUE TITLE
proparse error with def temp-table

### ISSUE THREAD

**Issue by darrenparr** [vscode-abl/vscode-abl#154]
I have several hundreds of proparse issues. All my source is successfully compiled. We use a temp table database connected with logical name temp-db.

DEF TEMP-TABLE ttRange LIKE TEMP-DB.ttRange

causes a proparse error. I suspect it can't handle the database name at all.

**Comment by darrenparr** [comment]
I specified a schemafile but not aliases. If I change the name from the name of the db to TEMP-DB it seems to work ok. Should I be putting TEMP-DB into the alias?

I thought that perhaps this should have been picked up from the -ld parameter but I guess that just the progress side of it and I imagine the proparse is a java process.

**Comment by gquerret** [comment]
Right, proparse is a Java process which doesn't know the -ld parameter. Changing the name attribute is the equivalent of -ld for proparse.

**Comment by darrenparr** [comment]
I did try changing just the alias this afternoon and that worked up until the point where I tried to open a file with the appbuilder. It failed there as it stated I couldn't have a logical database name the same as an alias.

I do love the product btw. I have all of my source being parsed which is no small task for our product as there are thousands of files. It's a pity I can't find a way of creating a task which inherits the propath from the project. It looks like I'm going to need an ini file to do this although I see you have a stub which sets the propath when you open the appbuilder etc.
